.github/workflows/ci.yml

```diff
@@ -98,6 +98,8 @@ jobs:
         run: yarn lerna run test-unit --scope=@sequelize/postgres
       - name: Unit tests (core - sqlite3)
         run: yarn lerna run test-unit-sqlite3 --scope=@sequelize/core
+      - name: Unit tests (sqlite3 package)
+        run: yarn lerna run test-unit --scope=@sequelize/sqlite3
```

Comment:
Do we really need to add this? since sqlite already represented by 

```
- name: Unit tests (core - sqlite3)
  run: yarn lerna run test-unit-sqlite3 --scope=@sequelize/core
```

Reply:
They run different tests. The scope `@sequelize/sqlite3` runs the tests in packages/sqlite3, while `@sequelize/core` runs the tests in packages/core